common/workunit/workunit.cpp

```diff
@@ -12962,6 +12962,16 @@ extern WORKUNIT_API void updateWorkunitTimeStat(IWorkUnit * wu, StatisticScopeTy
     wu->setStatistic(queryStatisticsComponentType(), queryStatisticsComponentName(), scopeType, scopestr, kind, description, value, 1, 0, StatsMergeReplace);
 }
 
+extern WORKUNIT_API void updateWorkunitNumStat(IWorkUnit * wu, StatisticScopeType scopeType, const char * scope, StatisticKind kind, const char * description, unsigned __int64 value, unsigned wfid)
```

Comment:
This isn't needed/correct.  The count is only used for keeping count of the number of instances of a statistic (and is generally not used.).

I would rename updateWorkunitTimeStat() above to updateWorkunitStat and call that.